src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -144,10 +144,9 @@ public DocWriteRequest<?> convertRecord(SinkRecord record, String index) {
         default:
           throw new DataException(
               String.format(
-                  "%s with key of %s and null value encountered (to ignore future records like"
+                  "%s has a null value (to ignore future records like"
```

Comment:
@snehashisp : we are still logging entire record .. ig we should avoid logging records too

Reply:
do you meant `recordString`. That only records the metadata of the record not the entire record. 
```private static String recordString(SinkRecord record) {
    return String.format(
        "record from topic=%s partition=%s offset=%s",
        record.topic(),
        record.kafkaPartition(),
        record.kafkaOffset()
    );
  }````